### ISSUE TITLE
US/Canada Based Maintainer Needed

### ISSUE THREAD

**Issue by dbrgn** [dbrgn/orochi#73]
8tracks are essentially blocking out all non-US and non-Canadian users starting today:

http://blog.8tracks.com/2016/02/12/a-change-in-our-international-streaming/

I asked whether this affects people based on their IP. It does:

> Yes, unfortunately this requires GeoIP blocking, as you mentioned, which means that content to all apps outside the US & Canada will be blocked. The YouTube streaming is a web integration using YT's API to call the same files that are included in an 8tracks playlist to stream from their servers when available.

This means that starting today, Orochi will cease to work outside the US or Canada.

8tracks started out as a likeable, very "different" streaming company. Their legal basis was that they could operate as an online radio by imposing some limits on playlists and playback.  It worked out great. They had a great app, a great website and friendly support staff. But it seems that politics and copyright / money have gotten in the way.

The following paragraph is part of their blogpost:

> We remain committed to bringing 8tracks to a global audience so all can join us in creating and discovering playlists for every taste, time and place.

To me, that's just empty marketing-speech. We need less barriers in the internet, not more! Country boarders should not interfere with the usage of the net.

---

**TLDR: If you're based in the US or in Canada and want to continue the development of Orochi, please let me know! I don't feel like investing more time into this project.**

**Comment by schmijos** [comment]
Thank you very much for your effort so far! The console client really meant something to me.

**Comment by dbrgn** [comment]
Note: So far, Orochi still works for me.

**Comment by mindstorms6** [comment]
@dbrgn I'm always around if you need a maintainer! I'll help in any way possible

**Comment by dbrgn** [comment]
@mindstorms6 cool, feel free to pick up or triage any open issues :)

**Comment by mindstorms6** [comment]
Can do!

**Comment by dbrgn** [comment]
Still works for me with a Swiss IP.
